External/FEXCore/Source/Interface/Core/Frontend.cpp

```diff
@@ -775,6 +775,10 @@ bool Decoder::NormalOpHeader(FEXCore::X86Tables::X86InstInfo const *Info, uint16
     const uint8_t Byte1 = ReadByte();
     DecodedHeader options{};
 
+    if ((Byte1 & 0b10000000) == 0) {
+      DecodeInst->Flags |= DecodeFlags::FLAG_REX_XGPR_R;
```

Comment:
According to documentation while in 32-bit mode this must ALWAYS be 1.
Maybe add an assert for 32-bit mode && 0 for this?